ckanext/oidc_pkce/templates/user/login.html

```diff
@@ -4,11 +4,11 @@
 #}
 
 {% block form %}
-    <p><a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-primary"><i class="fa fa-sign-in"></i> Log in via single sign-on</a></p>
+    <p><a href="{{ h.url_for('user.login') }}/oidc-pkce" class="btn btn-primary"><i class="fa fa-sign-in"></i> Sign in with your Digital ID</a></p>
```

Comment:
please update this to pull via a helper that can have the details altered via ckan.ini